FallHandling.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using BattleTech;
+
+namespace CharlesB
+{
+    public class FallHandling
+    {
+        private static readonly string KnockdownPhrasePath = Path.Combine(Core.ModDirectory, "phrases.txt");
+
+        /// <summary>
+        ///     displays a pithy floatie message over the supplied mech
+        /// </summary>
+        /// <param name="mech"></param>
+        public static void SaySomethingPithy(Mech mech)
+        {
+            if (!Settings.EnableKnockdownPhrases) return;
+            if (!mech.IsFlaggedForKnockdown) return;
+
+            var fileLoaded = false; // only initialize the list once, only if it's needed
+            var phrases = new List<string>();
+            var random = new Random();
+            if (!fileLoaded)
```

Comment:
I feel like you were trying to memoize here, but the the list is not saved to a static variable, so you are loading the file and parsing it every time this method is called.